src/utils/file-explorer.hpp

```diff
@@ -17,26 +17,22 @@ namespace pinocchio
    * @brief      Parse an environment variable if exists and extract paths according to the delimiter.
    *
    * @param[in]  env_var_name The name of the environment variable.
-   * @param[in]  delimiter The delimiter between two consecutive paths.
    *
    * @return The vector of paths extracted from the environment variable value.
    */
   PINOCCHIO_DLLAPI std::vector<std::string>
-  extractPathFromEnvVar(const std::string & env_var_name,
-                        const std::string & delimiter = ":");
+  extractPathFromEnvVar(const std::string & env_var_name);
 
 
   /**
    * @brief      Parse an environment variable if exists and extract paths according to the delimiter.
    *
    * @param[in]  env_var_name The name of the environment variable.
    * @param[out] list_of_paths List of path to fill with the paths extracted from the environment variable value.
-   * @param[in]  delimiter The delimiter between two consecutive paths.
    */
   PINOCCHIO_DLLAPI void
   extractPathFromEnvVar(const std::string & env_var_name,
-                        std::vector<std::string> & list_of_paths,
-                        const std::string & delimiter = ":");
+                        std::vector<std::string> & list_of_paths);
```

Comment:
I did not realize, but you break the API here. Would it possible to keep the former signature alive?

Reply:
I've restored the old API and also ported the delimiter change over